modules/gdnative/gdnative/packed_arrays.cpp

```diff
@@ -30,882 +30,101 @@
 
 #include "gdnative/packed_arrays.h"
 
-#include "core/variant/array.h"
-
 #include "core/variant/variant.h"
 
-#include "core/math/color.h"
 #include "core/math/vector2.h"
```

Comment:
It's weird that you'd still need `core/math/vector2.h` while the other ones are no longer used. What's the difference?

Reply:
Because of the Vector2i array. For the others I'm using the Packed* name which is defined in Variant, so I don't need the specific headers. Well, now that I think of it Variant also includes Vector2 so I could probably remove this anyway.